### PR TITLE
Update to latest library version from Packagist

### PR DESCRIPTION
**Pull request by felixarntz**
Now that [Third Party Capital is on Packagist](https://packagist.org/packages/googlechromelabs/third-party-capital), this PR updates to its latest version coming from there.

It also updates the readme file, adding an example for the recently added Google Tag Manager third party, and it adds a few other minor readme enhancements like linking to the JSON schema source files for additional information on the supported parameters. This is probably fine at this point, as from the JSON they're easy enough to understand for developers, and it means that we don't have to manually maintain documentation for all the fields in here.

### CODE REVIEW

**Comment by felixarntz** [comment]
@adamsilverstein Once this is merged, I think the codebase is ready for a 1.0 release, then only pending a first Packagist release of Third Party Capital. I believe the idea is to release its version 3.0 soon, and once that is out we can rely on it here and ship `wp-third-parties` 1.0.

File: README.md [hunk 1]
```diff
@@ -16,6 +18,22 @@ $ga = new Google_Chrome_Labs\WP_Third_Parties\Third_Parties\Google_Analytics(
...
+### Google Tag Manager
+
+```php
+$gtm = new Google_Chrome_Labs\WP_Third_Parties\Third_Parties\Google_Tag_Manager(
+	array(
+		'id' => 'GTM-...', // Replace this with your actual Google Tag Manager ID.
+	)
+);
+
+add_action( 'wp_loaded', array( $gtm, 'add_hooks' ) );
```
Comment (adamsilverstein) [inline README.md:34]: reviewing the entire doc I noticed they all include a hook line like `add_action( 'wp_loaded', array( $yte, 'add_hooks' ) );` except "Google Maps Embed". Should it also include the hook line? If not maybe a single line explaining why these are included in the other examples but not this one.

**Review by adamsilverstein — APPROVED** [review]
Looks good! One non-blocking question.

File: README.md (hunk 1 above)
Comment (felixarntz) [reply README.md:34]: @adamsilverstein Great point! I clarified this in https://github.com/GoogleChromeLabs/wp-third-parties/pull/3/commits/7c43462c962cfe6a5652ad523f29008dd2a5a005.

The reason Google Maps Embed doesn't need to add hooks is that there are no assets (scripts or stylesheets) needed for it. You could technically still include the same hook callback, but it just wouldn't do anything. So if you know that already, you can omit it, that's why the snippet here does that.

**Pull request merged**